### DISCUSSION TITLE
Does Kvrocks support multiple data centers?

### DISCUSSION THREAD

**Discussion by hongbo-miao** [apache/kvrocks#466] [Q&A] [answered]
I am doing some research about Redis solution supporting multiple data centers.
I found Ctrip 携程 has a [Redis solution](https://github.com/ctripcorp/x-pipe) supporting multiple data centers.

Then one of my friends recommends me Kvrocks!
Does Kvrocks support multiple data centers? I have read the README, but seems it didn't mention it.
Thanks! 😃

**Comment by git-hulk** [comment, marked as answer]
Thanks for your attention. Kvrocks don't support multiple DCs now. But @ShooterIT @Alfejik seems to have some thoughts and efforts on this issue.
